charts/kured/README.md

```diff
@@ -74,6 +74,8 @@ The following changes have been made compared to the stable chart:
 | `configuration.annotateNodes` | cli-parameter `--annotate-nodes`                                      | `false`                   |
 | `configuration.logFormat` | cli-parameter `--log-format`                                              | `"text"`                  |
 | `configuration.preferNoScheduleTaint` | Taint name applied during pending node reboot                 | `""`                   |
+| `configuration.preRebootNodeLabels` | Array of key-value-pairs to add to nodes before cordoning for multiple cli-parameters `--pre-reboot-node-labels` | `[]` |
```

Comment:
Should these new pre/post reboot config additions be present in main branch as well?

Reply:
Cool. I guess what I am observing is that other helm additions/changes during this release cycle already existed in main. Which flow is the preferable one? Helm changes landing on main prior to a release, or during the release process?

Reply:
I would prefer during the release-process, otherwise we would release a chart with flags which will cause kured to crash when they are enabled (cause the used image doesn't know these flags).
In the past for some changes there were PRs and we held them back until the release or if there were no PRs, I always implemented them in this "final PR".